src/pages/ProvideLiquidity/index.tsx

```diff
@@ -77,12 +79,10 @@ export default function ProvideLiquidity() {
             </RowBetween>
             <RowBetween>
               <TYPE.white fontSize={14}>
-                Add liquidity for our farmbots to support our ecosystem and earn liquidity provider fees!
+                Revo automatically stakes the liquidity that you provide, so that you can earn auto compounding rewards
```

Comment:
I think having a note here about how "zap-ins to farms are supported by liquidity pools; when you provide liquidity to support zap-ins, you'll earn _additional_ auto-compounding yield farming rewards on top of the rewards from the underlying farm" or something to that effect could be helpful. i also think the docs could use some reworking to really explain what this page does/how meta-farming rewards work... happy to tackle that.

Reply:
oh yes good idea, i will reword...